systemtest/src/test/java/io/strimzi/systemtest/bridge/HttpBridgeKafkaExternalListenersST.java

```diff
@@ -211,7 +211,7 @@ private void testWeirdUsername(ExtensionContext extensionContext, String weirdUs
             .withBootstrapAddress(externalBootstrapServers)
             .withNamespaceName(namespace)
             .withTopicName(topicName)
-            .withMessageCount(100)
+            .withMessageCount(MESSAGE_COUNT)
```

Comment:
Or maybe here should be the use value from the `TestStorage` instance, which I know is not instantiated in the test case but the aim is to have TestStorage in most of the tests where it make sense and in this one I think it does.